src/components/Statistics/PickupFeedback.vue

```diff
@@ -121,19 +132,30 @@ export default {
 
 <style scoped lang="stylus">
 .image-and-text
-  margin-bottom 3.5em
+  padding-bottom 1.5em
+  padding-top 1.5em
   .image-and-text-left
     width 30%
     max-width 10em
     margin auto
     padding 1em
   .image-and-text-right
-    width: 70%
+    width: 100%
+    padding 0 1em
     margin 0 auto
 .store-feedback
-  margin-top 2.5em
+  margin-top 2.5em !important
   .randomBanner
     display: block
     height: 26px
     overflow: hidden
 </style>
+
+<style lang="stylus">
+.q-field-dark.grey-font
```

Comment:
Please use a custom class to avoid matching unrelated components, e.g. `.pickup-feedback .q-field-dark .grey-font`. Unscoped CSS can get annoying...

Reply:
I actually didn't find a way to do it using scoped css, since quasar is creating the component there. The only way of changing the input field is using unscoped css somewhere I believe...

But I added a css class for pickup-feedback now and made it a subclass (I think that was also your suggestion)